cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,41 @@
+import CartPage from '../support/pages/cart.pageObject';
+import HomeAndCataloguePageObject from '../support/pages/homeСatalogue.pageObject';
+import ContactFormPageObject from '../support/pages/contactForm.pageObject';
+import PageObject from '../support/PageObject';
+
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+const homeAndCatalogue = new HomeAndCataloguePageObject();
+const cartPage = new CartPage();
+const contactForm = new ContactFormPageObject();
+const page = new PageObject();
 
+describe('Order flow', () => {
+  before(() => {
+    page.visit('https://www.demoblaze.com/');
   });
 
-  it('', () => {
+  it('should provide an ability to add the product to a cart and make a checkout', () => {
+homeAndCatalogue.clickOnCategory('Laptops');
+    homeAndCatalogue.clickOnProduct('Sony vaio i7');
```

Comment:
there is not function clickOnProduct() function in your homeCatalogue file. Check it pls